lib/components/chat_history/message.dart

```diff
@@ -141,7 +141,7 @@ class ChatHistoryMessage extends StatelessWidget {
     } else if (m is StreamStateEventModel) {
       return StreamStateEventWidget(m);
     } else if (m is TwitchFollowEventModel) {
-      return enableInlineEvents ? TwitchFollowEventWidget(m) : Container();
+      return TwitchFollowEventWidget(m);
```

Comment:
It's more robust to put the conditional here instead of in the follow event widget, and also use a `Selector`: https://pub.dev/documentation/provider/latest/provider/Selector-class.html

```
return Selector<EventSubConfigurationModel, FollowEventConfig>(
  selector: (_, model) => model.followEventConfig,
  builder: (_, config, _) => config.showEvent ? TwitchFollowEventWidget(m) : Container(),
);
```